certified-connectors/IN-D Payables/apiDefinition.swagger.json

```diff
@@ -1,3 +1,805 @@
+                          }
+                        },
+                        "description": "The line items in the Table is extracted (Validated)"
+                      }
+                    }
+                  },
+                  "description": "The Result Data for the whole batch of invoices (Validated)"
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "Authorization headers or API key not given",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Authorization headers or API key not given",
+                  "title": "Authorization error"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Invalid Request",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Invalid Request",
+                  "title": "Invalid Request"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal error",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Internal error",
+                  "title": "Internal error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Export validated data",
+        "description": "Export validated invoice data using particular batch id",
+        "operationId": "ExportValidatedData",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "batch_id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "validated invoice batch data",
+                  "title": "batch id"
+                }
+              },
+              "required": [
+                "batch_id"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.in-d.ai"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.in-d.ai/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;IT Operations"
+    }
+  ],
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "x-api-key"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": []
+}
```

Comment:
@gowthamd3 Please remove below swagger code.

Reply:
@sriyen-msft  removed the code you have mentioned

Reply:
@gowthamd3 It is still present in the swagger. Please remove it

Reply:
@sriyen-msft  i couldn't find any empty lines, pls let me know at which row empty lines are present?.